api-samples/declarativeNetRequest/no-cookies/README.md

```diff
@@ -0,0 +1,11 @@
+# chrome.declarativeNetRequest - No Cookies
+
+This sample demonstrates how to use the [chrome.declarativeNetRequest](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/) API to remove the "Cookie" header from requests.
+
+## Overview
+
+Once installed, any requests made in the main frame will be sent without the "Cookie" header.
```

Comment:
I meant restrict this example to a single web site. I when I was testing this, I forgot it was installed couldn't figure out why web page were behaving so weirdly. I didn't restrict it to Google sites because, as you know, enterprise policy prevents extensions from working on our own sites. I didn't use example.com because it doesn't use cookies. 

If we can't come up with a site where we can do this safely, it be worth adding a note to be careful.

Reply:
I spent so long thinking about this. I settled on `"urlFilter": "|*?no-cookies=1"`, which will work for any URL that ends in `?no-cookies=1`. I think that's a nice way to make this work everywhere while reducing the overall risk.